canonical/ubuntu.com
# feat: /download/arm 24.10 copy update

## Done

- Applied copy update as per [doc](https://docs.google.com/document/d/1whVU3DvB9j5k6Ed3UvLZCEN6BU-lZcow4ZefqKQqokg/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14382.demos.haus/download/arm
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that link has been added

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14864

## Screenshots

![image](https://github.com/user-attachments/assets/b43ff501-ae72-4850-ae34-a3388fa29adc)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
